dot/parachain/types/types.go

```diff
@@ -740,6 +747,19 @@ type Subsystem interface {
 	Stop()
 }
 
+// UpgradeRestriction a possible restriction that prevents a parachain
+// from performing an upgrade
+// TODO: should be scale encoded/decoded
```

Comment:
Will this be addressed in this PR or should an issue be made?

Reply:
addressed here